vtex/ab-tester
# Remove confidence level

#### What is the purpose of this pull request?
Remove the use of the confidence level inputed by the user.

#### What problem is this solving?
We don't need anymore to request  the desired confidence level and then we can remove all its uses.

#### How should this be manually tested?
Call `initialize` and `status` and verify if `ab-tester` works properly, i.e. initialize test correctly and return response for `status` correctly.

#### Types of changes
